packages/github/package.json

```diff
@@ -38,7 +38,8 @@
   },
   "dependencies": {
     "@octokit/graphql": "^4.3.1",
-    "@octokit/rest": "^16.15.0"
+    "@octokit/rest": "^16.15.0",
+    "@octokit/webhooks": "^7.0.0"
```

Comment:
what is the size impact to consumers of the `github` module?

Reply:
None of the code gets routed to ncc, so there is no impact for consumers using that.

However, I think we can install this as a dev dependency, so users doing npm install --prod won't even download it. Going to test that out and make that corresponding change.

Reply:
There's a bit of a mixed reaction in the ts/js community, but it looks like we shouldn't install as dev-dependencies [we export these types](https://stackoverflow.com/a/46011417) [open @types issue](https://github.com/Microsoft/types-publisher/issues/81)

In its current state (saved as a dependency)
Users who upload their production npm packages rather then using ncc are going to have the size of their  action increased by the size of the npm package and its dependencies, which is 291 KB or 400kb on disk for my machine.